lib/wasi/src/lib.rs

```diff
@@ -838,6 +866,11 @@ fn mem_error_to_wasi(err: MemoryAccessError) -> Errno {
     }
 }
 
+fn mem_error_to_wasi_error(err: MemoryAccessError) -> WasiError {
```

Comment:
`From<MemoryAccessError> for WasiError`

Reply:
But this seems like it should be a `WasiRuntimeError`.

Reply:
The function wasn't used anymore anyway so I deleted it